### PR TITLE
Added "Handling PRs Made Without Being Assigned" section at "layer5/s…

### PR DESCRIPTION
**Pull request by saurabh100ni**
### Description
By taking inspiration from [conversation on Slack](https://layer5io.slack.com/archives/C015QJKUMPU/p1694272281765699) about **handling PRs made without being assigned** I tried to make it more understandable by updating the [contribution page](https://layer5.io/community/handbook/contribution) of our [Community Handbook](https://layer5.io/community/handbook/) by adding a section for the same.

image

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit db8c858120c1d10ea4d378d67e31f74825ba0ed9 at: https://64fcd53cece3552863c202e4--layer5.netlify.app

**Comment by Ghat0tkach** [comment]
hey @saurabh100ni 

Let's discuss it on the websites call.
Please add this as an agenda item in the meeting minutes, if you would :)
https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima

**Comment by saurabh100ni** [comment]
sure thing! 🙌

**Comment by l5io** [comment]
🚀 Preview for commit 4c3cd68eb21289b433b4b02096a98d60ec1fc711 at: https://650757c7e988626168e70999--layer5.netlify.app

**Comment by saurabh100ni** [comment]
@ritiksaxena124
can you review the PR?

File: src/sections/Community/Handbook/contributing.js [hunk 1]
```diff
@@ -228,6 +228,128 @@ const contributingGuide = () => {
...
+                    <br/><strong>Example:</strong> <a href="https://github.com/meshery/meshery/issues/8757">[MeshModel] [UX]: Icon for Kubernetes Component - Endpoints #8757</a>
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    <strong>Requesting Reassignment:</strong> If you are interested in working on an issue that is already assigned to someone else, it's important to respect their efforts and contributions. First, politely reach out to the assignee and express your interest in collaborating on the issue. Discuss the possibility of working together or sharing the workload. If the assignee agrees or is open to collaboration, you can proceed. If they are unable or unwilling to collaborate, and the issue aligns with your skills and interests, kindly request reassignment from the maintainers, providing a clear and courteous explanation for the request.
```
Comment (ritiksaxena124) [inline src/sections/Community/Handbook/contributing.js:346]:
```suggestion
                    <strong>Requesting Reassignment:</strong> If you are interested in working on an issue that is already assigned to someone else, it's important to respect their efforts and contributions. First, politely reach out to the assignee and express your interest in collaborating on the issue. Discuss the possibility of working together or sharing the workload. If the assignee agrees or is open to collaboration, you can proceed. If the assignee is not responsive or there has been no progress on the issue for a long time and the issue aligns with your skills and interests, kindly request reassignment from the maintainers, providing a clear and courteous explanation for the request.
```

**Comment by l5io** [comment]
🚀 Preview for commit 137bba4aa828b9bbe059e07c9aeafe948b3d8a96 at: https://650c0637c2e4f7027388552e--layer5.netlify.app

**Comment by saurabh100ni** [comment]
@ritiksaxena124 👀

**Comment by ritiksaxena124** [comment]
LGTM 👌

**Comment by saurabh100ni** [comment]
@ritiksaxena124 can you merge?

**Pull request merged**